### ISSUE TITLE
Publish release?

### ISSUE THREAD

**Issue by wezm** [rabite0/hunter#1]
This looks like a really well built tool. I'd love to create and [AUR](https://aur.archlinux.org/) package for hunter. It's best to build these against published releases instead of straight from git master though. I would be great if you could tag a [release](https://help.github.com/en/articles/creating-releases).

**Comment by rabite0** [comment]
Thank you!
I meant to do that anyway, but forgot. I shamelessly tagged this with 1.0 now, does that work for you?

**Comment by cpdean** [comment]
Also it looks like you haven't made a [crate of this yet](https://crates.io/search?q=hunter).  if you do a `cargo publish` then everyone can install this with `cargo install hunter`.

**Comment by rabite0** [comment]
Oh, yes, you're right. Will do that when I get home.

**Comment by rabite0** [comment]
Done.

**Comment by wezm** [comment]
Thank you. It looks like someone beat me to publishing an AUR package:
https://aur.archlinux.org/packages/hunter/

It looks like hunter requires nightly Rust. It would be great if you added a [rust-toolchain](https://github.com/rust-lang/rustup.rs#the-toolchain-file) file to the root of the repo with the version you're using so people can replicate that version and guarantee a successful build.

**Comment by rabite0** [comment]
Thank you too, your suggestions are really appreciated. Sorry to hear, that someone snatched the packaging it under your nose, but thanks to that person, too! :)

I added a rust-toolchain file, like you said, someone had problems compiling hunter, probably because of a too old nightly compiler. Sadly, there doesn't seem to be a way to just specify a minimum version, but it'll do. I hope I did this right. Thanks again.
